python-lib/ocr_constants.py

```diff
@@ -1,5 +1,5 @@
 class Constants:
```

Comment:
I would rename all the `ocr_xxxx.py` python lib files to `text_extraction_xxxx.py`

Reply:
I changed the file structure in [aca1dc4](https://github.com/dataiku/dss-plugin-tesseract-ocr/pull/71/commits/aca1dc4ac72e7eb4b8c4061bbb425bc999cf48e7) to have modules for text_extraction, ocr and utils